Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM httpd:latest
+
+# Copy configuration file for apache
+COPY ./apache-httpd.conf /usr/local/apache2/conf/httpd.conf
+
+# Expose apache
+EXPOSE 80
+
+# Copy the project files into place.
+COPY public /usr/local/apache2/htdocs/public
```

Comment:
So make me understand: all you need is your static HTTP and JS files in the public root and it works? You are using no router? I thought I saw something like http://localhost:port/some/route today in the presentation, and this `/some/route` works with Apache this way? I'd be surprised that there is no need to run a backend server to catch calls and re-route them the right way (or better: render the right page). If Apache does that: great. But it just seems fishy to me